libbeat/beat/beat.go

```diff
@@ -252,6 +254,11 @@ func (b *Beat) launch(bt Creator) error {
 		return err
 	}
 
+	err = b.loadTemplate()
```

Comment:
TBH this is a deal breaker for me, especially the part where Beats don't start if ES is not up is going to really annoying to users. 

Ideally we'd also have the pipelines loaded on each connection. It's currently done only on startup, but that can lead to errors similar to these. I think we need some sort of generic callback mechanism to do this.